tensorflow/lite/kernels/pooling_test.cc

```diff
@@ -12,8 +12,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
-#include <cstdarg>
 #include <gtest/gtest.h>
+#include <cstdarg>
```

Comment:
Why is this order changed?

Reply:
@jianlijianli : This is because of static code check warning modified, will revert it back.